### PR TITLE
Add bias state to ioc, update scripts as needed

### PR DESCRIPTION
**Pull request by ihumphrey**

### CODE REVIEW

**Comment by birdzix** [comment]
would i need to update `auto_power_down_script.py` to write to the `BiasOff` PV?

File: fastccd_support_ioc/fastccd_support_ioc.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from caproto import ChannelType
 
 from . import utils, pvproperty_with_rbv, wrap_autosave, FastAutosaveHelper
+from utils import setClocksBiasOff, setClocksBiasOn
```
Comment (ronpandolfi) [inline fastccd_support_ioc/fastccd_support_ioc.py]: These modules are written as scripts; importing them executes their action. See how the autostart is invoked for a better approach.

File: fastccd_support_ioc/fastccd_support_ioc.py [hunk 2]
```diff
@@ -244,6 +246,53 @@ async def Initialize(self, instance, async_lib):
...
+                elif value == "Powering Off...":
+                    value = await self._power_off_bias(None, None)
+
+            return value
+
+        async def _power_on_bias(self, instance, value):
+            async with com_lock:
+                power_check_no_bias_clocks()
+                setClocksBiasOn()
+                await self.async_lib.library.sleep(1)
```
Comment (ronpandolfi) [inline fastccd_support_ioc/fastccd_support_ioc.py]: This wait needs to be greater than 3 sec to ensure that the values read back in the check are updated.

Comment (birdzix) [reply fastccd_support_ioc/fastccd_support_ioc.py]: where does that > 3 second requirement from from? something to do with powering on the bias or something related to caproto/EPICS?

File: fastccd_support_ioc/fastccd_support_ioc.py (hunk 1 above)
Comment (birdzix) [reply fastccd_support_ioc/fastccd_support_ioc.py]: ah right -- i needed to access them via the utils.scripts module

**Review by ronpandolfi — APPROVED** [review]

**Pull request merged**